extensions/tokenizers/rust/src/layers/cublaslt.rs

```diff
@@ -5,29 +5,23 @@ use std::sync::Once;
 #[cfg(feature = "cuda")]
 use candle_cublaslt::{fused_batch_matmul, fused_matmul, Activation, CublasLt};
 
-static INIT: Once = Once::new();
-static mut CUBLASLT: Option<CublasLtWrapper> = None;
-
-pub fn get_cublas_lt_wrapper() -> Option<&'static CublasLtWrapper> {
+pub fn get_cublas_lt_wrapper(device: &Device) -> Option<CublasLtWrapper> {
```

Comment:
Why static initialization cause issue?

Reply:
Because static cublaslt is created on device 0, if device 1 use it, it will cause illegal memory access. This is causing the integration test failure now.